src/core/server/saved_objects/import/import_saved_objects.test.ts

```diff
@@ -109,14 +121,17 @@ describe('#importSavedObjectsFromStream', () => {
       typeRegistry,
       namespace,
       createNewCopies,
+      dataSourceId,
     };
   };
-  const createObject = (): SavedObject<{
+  const createObject = (
+    dataSourceId: string | undefined
```

Comment:
Provide default value here as `undefined` to avoid change in existing test cases.

Reply:
Nice catch, thanks for the suggestion, updated